### PR TITLE
Create app bar

### PR DESCRIPTION
**Pull request by lambo94**
Static nav bar with 5 options. Added CLSX and also added MUI and MUI Icons like we discussed previously.

### CODE REVIEW

File: src/components/Layout/AppBar.tsx [hunk 1]
```diff
@@ -1,41 +1,49 @@
...
 
 //Probably could have this populate from the DB but its static so...
 const links = [
-    { name: 'Home', href: '/home', icon: 'HomeIcon' },
-    { name: 'Meta Hub', href: '/meta', icon: 'MetaIcon' },
-    { name: 'Creators', href: '/creators', icon: 'CreatorsIcon' },
-    { name: 'TTK Tool', href: '/ttk', icon: 'TtkToolIcon' },
-    { name: 'Hub', href: '/hub', icon: 'HubIcon' },
+    { name: 'Home', href: '/', icon: HomeRoundedIcon },
+    { name: 'Meta Hub', href: '/meta', icon: CarpenterRoundedIcon }, // There's no icon in MUI for a gun :P
```
Comment (KelvinJardin) [inline src/components/Layout/AppBar.tsx:17]: ahh guessing no gun in there XD

File: src/app/layout.tsx [hunk 2]
```diff
@@ -21,6 +22,7 @@ export default function RootLayout({
           <NextAuthProvider>
                 {children}
           </NextAuthProvider>
+          <AppBar />
```
Comment (KelvinJardin) [inline src/app/layout.tsx:25]: We probably should put it inside the auth provider

noticed the app bar is customizable per user

**Review by KelvinJardin — CHANGES_REQUESTED** [review]
Loooks sick! 
Wondering how long you took to land on using the carpenter saw as a gun replacement? 😆 

just one smoll change

File: src/components/Layout/AppBar.tsx (hunk 1 above)
Comment (lambo94) [reply src/components/Layout/AppBar.tsx:17]: Nah, felt like you'd appreciate a saw as replacement haha

**Review by KelvinJardin — APPROVED** [review]
Siiick, love the wave/pulse animation on the buttons 😍

**Pull request merged**